recipes/sdl_ttf/all/patches/cmake-fix-link-target-2.0.15.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ed6e249..e67a40b 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -22,6 +22,14 @@ set(SDL_TTF_VERSION "${SDL_TTF_MAJOR_VERSION}.${SDL_TTF_MINOR_VERSION}.${SDL_TTF
+ 
+ ##### library generation #####
+ add_library(SDL2_ttf SDL_ttf.c SDL_ttf.h)
++
++if(TARGET SDL2::SDL2)
++  target_link_libraries(SDL2_ttf SDL2::SDL2 Freetype::Freetype)
++else ()
++  target_link_libraries(SDL2_ttf SDL2::SDL2-static Freetype::Freetype)
++endif ()
++
++
+ target_link_libraries(SDL2_ttf SDL2::SDL2 Freetype::Freetype)
```

Comment:
This patch is not correct

Reply:
Sorry for my misunderstanding.

Is following condition is right in 2.0.15?
```
if (BUILD_SHARED_LIBS)
```

Reply:
`target_link_libraries(SDL2_ttf SDL2::SDL2 Freetype::Freetype)` must be removed and replaced by the block you have added.

Reply:
Oh. I miss line 17.
Thank you!